source/_integrations/squeezebox.markdown

```diff
@@ -56,6 +57,17 @@ transporter_toslink:
 - **Needs restart**:  Server Service needs to be restarted (typically, this is needed to apply updates).
 - **Library rescan**:  The music library is currently being scanned by LMS (depending on the type of scan, some content may be unavailable).
 
+### Sensors
+
+- **Last scan**: Date of the last library scan.
+- **Player count**: Players on the service.
+- **Player count off service**: Players not on this service.
+- **Total albums**: Albums currently in the service.
+- **Total artists**: Artists currently in the service.
+- **Total duration**: Duration of all tracks in service.
```

Comment:
What is the time format used here? Minutes and seconds, MM:SS? Hours, minutes, and seconds?

Reply:
So its seconds but duration auto scales, to display SS:MM:HH on larger collections

Reply:
thank you for clarifying. Just to be sure about the order: it's  SS:MM:HH, and not HH:MM:SS ?

Reply:
you are correct I transposed it in my comment 1433:35:25 on my prod system.